.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
```

Comment:
- [ ] **Note that your project can not be approved  if you do not implement the request changes** :crying_cat_face: 
- [ ] As suggested by the previous reviewer, Kindly ensure that the test.md file is present in the ```.gitignore``` file. It's a part of the project requirement. :100: